src/Model/Form/RegisterFormModel.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
I don't know if the folder is correctly named and is the file name is adequate, might need some feedback on that

Reply:
For every from's classes you can move the folde 'Form' at src root and you can rename your file by removing 'Model'

you will : 

src/Form/RegisterForm.php